R/utils.R

```diff
@@ -465,7 +468,8 @@ average_biological_replicates_dt <- function(
   
   average_fields <- setdiff(names(Filter(is.numeric, data)), c(unlist(pidfs), var, iso_cols))
   geometric_average_fields <- intersect(geometric_average_fields, names(dt))
-  group_by <- setdiff(names(data), c(average_fields, var, id_cols, "fit_type", "Fit Type"))
+  fit_type_average_fields <- intersect(fit_type_average_fields, names(dt))
```

Comment:
GJ, can we add a simple test to prevent this issue in the future?

Reply:
done in f053c6bcb07c592848e59a619e2bef80a8cbade4.